### DISCUSSION TITLE
Mesh Creation problem

### DISCUSSION THREAD

**Discussion by fysis88** [Jyers/Marlin#1112] [Q&A] [answered]
Hello. 
First of all , congratulations for the great firmware.
i am facing some issues , regarding the Mesh creation . 
I am using the UBL-BLtouch 10x10 edition of the firmware. 
**_My first question is_** : When i select "Create new mesh" , the machine tilts only 62/100 points. Is that normal?
After that , i save , and check the mesh at the "mesh viewer".
So far so good...but after a while , or maybe after a print , the mesh looks different , at the viewer.
**_Should i select_** "autotilt current mesh' , first?
**_What is the use of_** "convert current mesh to plane" ?
**_Is my starting gcode correct?_** `G92 E0 ; Reset Extruder
G28 ; Home all axes
G29 A ;
G29 L0; Load Saved Mesh
G29 J2; Probe 4 Points and Adjust the mesh`

Hope not to get tired by my questions :) :) :

**Comment by gothcha** [comment, marked as answer]
Hi, yes 62/100 is normal as some points are not reachable due to probe offsets. After a print the mesh has been tilted so can look slightly different as the tilt points can potentially not match exactly with the original mesh points. This has been discussed at length in an earlier thread #982 The convert mesh to plane is not needed in normal use so ignore it. Your start code looks fine,  it it may be worth trying G29 J to use the three point tilt and see if this works better (three points are perfectly adequate to calculate the plane).
You can autotilt the mesh on the display and check that the mesh still looks reasonable, you can even then save the tilted mesh to EEPROM if the bed has changed.

Hope this helps enjoy!
